### ISSUE TITLE
Suggestion : Facilitate cross-community projects visibility

### ISSUE THREAD

**Issue by Siltes** [animo/awesome-self-sovereign-identity#24]
Hello, 

I would like in the first place to thank you to have set up this repo.

Considering all the new projects related to SSI, I was wondering if there might be a possibility of refactoring the "Technical Specifications" and the "Tools & Technologies" sections in order to **better reflect the different communities as well as the cross-community projects within the SSI landscape**. 

I got this idea after encountering [this simplified map](https://github.com/decentralized-identity/decentralized-identity.github.io/blob/master/assets/map-of-adjacent-orgs-and-specs--sept-2020--one-sided.pdf) and [this detailed mapping](https://github.com/decentralized-identity/decentralized-identity.github.io/blob/master/assets/crosscommunity-architecture-survey-oct-2020.pdf) on the DIF website and found it **really** helpful to have a complete overview of all the things graviting around SSI.

Also, maybe it's unrelated, but does anyone know some markdown plugins in order to maintain such kind of knowledge base (in this particular form)? I really found that super useful but I guess it takes an awful lot of time to keep it updated (reason why it has been updated back in oct 2020)... I guess mermaidJS could be used; [Markmap](https://markmap.js.org/) could be also an option but you would loose one dimensionality.

Best regards,
Edoardo

**Comment by anagoessens** [comment]
I think this is a very good idea. Especially the relations between the projects (the simplified map you shared) is great. It's difficult though, I'm very interested in how we can create something that is:

1. Accessible to beginners to the space
2. Complete / complete enough
3. Not too difficult to keep up to date. 

Personally, I'd really like this to be very accessible to interested people with very little in depth knowledge. But I don't know if that's really the target audience in general. Do you have a mock-up of what the refactor would look like? I'm interested to explore this further and willing to spend some time on it as well.

**Comment by Siltes** [comment]
We're on the same page!

The only point that I would like to discuss is your definition of **accessibility to beginners to the space**. I think that the SSI topic is by definition very technical once you want to go into the specs (section I wanted to refactor), being currently discussed and re-defined; hence, the target would be in any case people with a technical background (I suppose). In any case, different sections of the document can address different needs and kind of audiences, that is why I was actually focusing on the more technical ones: if one is checking these sections is that he's already a step over "little depth in knowledge".

In any case, what I had in mind is first to reflect the 3 main horizontal stacks within both the Technical Specs & Tools and Technologies sections, then sort by kind of working community.

This would look like something like this (I'll take the current links that are present in the Technical Specs as an exemple to refactor):

## 📄 Technical Specifications

Here you will find an overview of the technical specifications, grouped by level and sorted by community. 

### Verifiable Credentials / Trusted Data Storage & Exchange

#### W3C

- [W3C Verifiable Credentials](https://www.w3.org/TR/vc-data-model/) - Specification of verifiable credentials (VCs).
- [W3C Verifiable Credentials Implementation Guidelines](https://www.w3.org/TR/vc-imp-guide/) - Implementation guidelines for verifiable credentials.

#### DIF

- [DIF Presentation Exchange](https://identity.foundation/presentation-exchange/) - Credential format and transport agnostic specification to articulate proof requirements and submit proofs.

- [DIF Credential Manifest](https://identity.foundation/credential-manifest) - Common data format for describing the inputs a subject must provide for credential issuance.

#### Hyperledger Foundation

- [Hyperledger Indy Improvement Proposals](https://github.com/hyperledger/indy-hipe/blob/master/index.md) - Specifications of Indy protocols and standards.

### DIDComm and Sub-protocols

#### DIF

- [DIF DIDComm Messaging](https://identity.foundation/didcomm-messaging/spec) - A secure, private communication methodology built atop the decentralized design of DIDs.

### DIDs, Anchors and Registries

#### W3C

- [W3C Decentralized Identifiers](https://www.w3.org/TR/did-core/) - Specification of decentralized identifiers (DIDs).

#### DIF

- [Key Event Receipt Infrastructure (KERI) collected resources](https://keri.one/keri-resources/) - Collected list of resources on Key Event Receipt Infrastructure.

---

Concerning **completeness and update difficulty**, I think that it depends solely on the contributions on this repo, even if I think there's some interest in the community currently to clarify some aspects and better understand who works on what... It could be interesting to have some insight from CCG/DIF/Hyperledger members maybe? Or if we wanted to go full automata a scraper which detects new elements on some target repos/websites could be the way to go but I think I am going too far here...

In any case, thanks for your time and looking forward to have your feedback on my proposition.

---

P.S.:

Something that can be also added to provide clarity could be a list of **main actors and OS communities** working on these subjects with a scope of action (which would also help to understand the different WGs); it could take a while to map it but the update rate here would be surely lower.

Also, a reordering could be done in the "Products and Services", to better differentiate Wallets, Agents, APIs and other kinds of platforms; the current segmentation being: projects, identity wallets, software development kits (which should be in tools?), and notable identity products and services (where you'll find again a link to Evernym and Tykn Products that themselves contain a Wallet solution). But this can be a different thread of discussion.

**Comment by anagoessens** [comment]
Do you have time to turn this into a PR? I think the idea is sound, and we should just get to work implementing and commenting in that form. 

I'll start on a PR for the products and services segment in the meantime, as it has been bothering me for ages. The problem is that for a while it was more specified, but we had to create more and more categories as companies often offer multiple products (a cloud api, wallet, end-to-end platform, cloud wallet, mobile sdk,  etc.). It ended up basically being 6 categories with the same companies mentioned below them. 

But I think we can keep it simple and informative, while still allowing for multiple categories.

**Comment by Siltes** [comment]
Yep, I'm on it, I'll open it by the end of the week! Looking forward to your PR as well.
Shall I leave the issue open for the moment?
